### ISSUE TITLE
Callout - use of aria-label in <p>

### ISSUE THREAD

**Issue by HerinHentry** [govau/uikit#408]
Regarding callout https://designsystem.gov.au/components/callout/

The use of aria-label in the `<section>` tag adds value as it adds a heading to the container. If you are looking at the element list using screen reader the aria-label added will be listed out.

But, for a `<p>` tag, I am not sure if aria-label adds any value. 

example:
```
<p class="au-callout" aria-label="Callout description1">
  A callout.
</p>
```

further reference: https://www.w3.org/TR/html51/grouping-content.html#elementdef-p

**Comment by TrebBrennan** [comment]
I'm assuming you're referring to this example?
```
<p class="au-callout" aria-label="Callout description1">
  A callout.
</p>
```

I think the actual intent is that it could be used more like this:
```
<p class="au-callout" aria-label="quote">
  "This is a quote"
</p>

<p class="au-callout" aria-label="example">
  This is an example
</p>

<p class="au-callout" aria-label="tip">
  This is a tip
</p>
```
Mostly so that any extra meaning isn't only visual.

**Comment by HerinHentry** [comment]
I think screen reader does not read the aria-label, at least NVDA I am checking with. So, the non-visual content does not add value. (maybe I'm wrong as I do not have access to another SR to check now)

**Comment by TrebBrennan** [comment]
I can confirm the `aria-label` is read by macOS-VoiceOver.
https://designsystem.gov.au/components/callout/accessibility/#screen-readers

However I'm going to recommend an update to the documentation to try and clarify all this a bit better.

**Comment by theartofpeople** [comment]
Hi - just wondering about the aria-label on a <p> tag - does this run the risk of cloaking the contents as described at https://www.w3.org/TR/wai-aria-practices-1.1/#no_aria_better_bad_aria ?

**Comment by alex-page** [comment]
Its a great point @theartofpeople I'll get this investigated in the few weeks.

**Comment by theartofpeople** [comment]
Kia ora @alex-page - thanks :-) 
Also since commenting, I've been told by our accessibility consultants that an aria-role should be used, since paragraphs have no default role. We didn't get any feedback about cloaking the content.

**Comment by alex-page** [comment]
Thanks again for checking back in - sorry for the late response before. 

Really useful information I will run some different paragraphs through my screen readers and report back. Hoping to pick this up in the next sprint.

**Comment by theartofpeople** [comment]
Hey no worries. If its any help, we're using:

`<div role="note" aria-label="Important">
<p>...</p>
</div>`

With the understanding that support should improve in future (eg not currently supported by NVDA)

**Comment by theartofpeople** [comment]
(Trying again as code was stripped...)

div role="note" aria-label="Important"
   paragraph etc
end div

**Comment by alex-page** [comment]
You can try putting the code inside three ` and ending with three `

**Comment by alex-page** [comment]
Thank you for all the feedback, the best course of action is to remove the `aria-label` all together. We can then recommend having a title inside the callout.

Here is a great article around `aria-label` and how it can be incorrectly used:
http://www.heydonworks.com/article/aria-label-is-a-xenophobe

If the user needs to hide it they can use the utility mixin `@include AU-sronly()` to hide the title so only screen readers can see it.

As of now I am updating the documentation and tests to use the following markup:
```
<section class="au-callout">
  <h2 class="au-callout__title">Description of callout</h2>
  <p>Hello world some lorem ipsum here</p>
</section>
```

If you have any feedback or want to recommend any changes happy to talk here!
